src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -100,15 +100,13 @@ class SpanDetail extends React.Component<Props, State> {
       field: ['transaction', 'id', 'trace.span'],
       sort: ['-id'],
       query: `event.type:transaction trace:${traceID} trace.parent_span:${spanID}`,
-      project: organization.features.includes('global-views')
-        ? undefined
-        : [Number(event.projectID)],
+      project: [] as number[],
```

Comment:
is empty array fine? instead of undefined? in that case can you do

```js
 project: organization.features.includes('global-views') ? [] : [Number(event.projectID)]
``` 
?